bundles/framework/divmanazer/instance.js

```diff
@@ -181,11 +183,30 @@ Oskari.clazz.define('Oskari.userinterface.bundle.ui.UserInterfaceBundleInstance'
             this.sandbox.unregister(this);
             this.started = false;
         },
+        // TODO: rename "getDefaultFlyoutPosition()"?
         getMapdivOffset: function () {
-            var mapdiv = jQuery('#mapdiv');
+            const dimensions = getNavigationDimensions();
+            if (!dimensions) {
+                // no nav
+                return {
+                    top: 0,
+                    left: 0
+                };
+            }
+
+            let left = 0;
+            let top = 0;
+            if (dimensions.left === 0) {
```

Comment:
dimensions now has `placement`. Use that instead to determine left. top. right, bottom.